frontend/src/container/AlertHistory/Statistics/StatsCard/StatsCard.tsx

```diff
@@ -88,13 +101,13 @@ function StatsCard({
 				/>
 			</div>
 
-			{!isEmpty && (
-				<div className="stats-card__graph">
-					<div className="graph">
-						<DotChartOutlined className="graph-icon" />
-					</div>
+			<div className="stats-card__graph">
+				<div className="graph">
+					{!isEmpty && timeSeries.length > 1 && (
```

Comment:
can timeSeries be undefined ?

Reply:
I have added a default value of `[]` to `timeSeries` prop. even if it's undefined this won't break IMO.